django/core/management/commands/sqlmigrate.py

```diff
@@ -34,7 +34,8 @@ def handle(self, *args, **options):
         connection = connections[options['database']]
 
         # Load up an executor to get all the migration data
-        executor = MigrationExecutor(connection)
+        # Do not replace migrations
+        executor = MigrationExecutor(connection, replace_migrations=False)
```

Comment:
I think we can make small refactoring in advance. I would move `MigrationExecutor.collect_sql()` (which is used only by `sqlmigrate`) to the `MigrationLoader`. With this change we'll be able to use here `MigrationLoader` and simplify changes.